pkg/api/v1alpha1/snowmachineconfig_types.go

```diff
@@ -17,6 +17,7 @@ type PhysicalNetworkConnectorType string
 type SnowInstanceType string
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// Important: Run "make generate" to regenerate code after modifying this file
```

Comment:
should we test this in a pre-submit?

Reply:
Created this issue: https://github.com/aws/eks-anywhere/issues/2734